compiler/src/dotty/tools/dotc/parsing/xml/MarkupParsers.scala

```diff
@@ -371,10 +371,17 @@ object MarkupParsers {
         // parse more XML?
         if (charComingAfter(xSpaceOpt()) == '<') {
           while {
-            xSpaceOpt()
-            nextch()
-            ts.append(element)
```

Comment:
I don't understand why my previous fix https://github.com/lampepfl/dotty/commit/624a6e08619b725cd32b1bade06d6c814225cd0f used `ts.append(element)` instead of `content_LT(ts)` which is what scala 2 has. (I would expect the codebases not to diverge much.)

Reply:
I don't know why too, but I suspect it was just a mistake as `element` cannot consume XML special nodes.

Reply:
By the way, current Scala 2 also uses content_LT instead of element
https://github.com/scala/scala/blob/8d598d102f0c2f23616ccd732e3ae93765da497a/src/compiler/scala/tools/nsc/ast/parser/MarkupParsers.scala#L392

Reply:
Yes, that is what I was comparing. oh, my previous comment says just that.